main.nf

```diff
@@ -733,11 +703,9 @@ process assign_clones{
     tag "${id}" 
     publishDir "${params.outdir}/define_clones/$id", mode: 'copy',
         saveAs: {filename ->
-            if (filename.indexOf(".fasta") > 0) "fasta/$filename"
-            else if (filename.indexOf(".log") > 0) null
-            else if (filename.indexOf("table.tab") > 0) "info/$filename"
-            else if (filename.indexOf(".tab") > 0) "table/$filename"
-            else if (filename == "command_log.txt") "$filename"
+            if (filename.indexOf("table.tab") > 0) "$filename"
+            else if (filename.indexOf("clone-pass.tab") > 0) "$filename"
```

Comment:
A lot of different cases of if statements in your code, I wonder if that could be done in an easier way. But if it works great! ;)